.github/workflows/base-ci-goreleaser.yaml

```diff
@@ -6,13 +6,19 @@ on:
       distribution:
         required: true
         type: string
+      goos:
+        required: true
+        type: string
+      goarch:
+        required: true
+        type: string
 
 jobs:
   check-goreleaser:
     strategy:
       matrix:
-        GOOS: [ linux, windows, darwin ]
-        GOARCH: [ "386", amd64, arm64, ppc64le, arm, s390x ]
+        GOOS: ${{ fromJSON( inputs.goos) }}
+        GOARCH: ${{ fromJSON( inputs.goarch) }}
```

Comment:
This change (and the one in base-release), could be done in a separate PR if desired.